applications/ExternalSolversApplication/external_includes/feast_solver.h

```diff
@@ -319,7 +319,13 @@ class FEASTSolver: public LinearSolver<TSparseSpaceType, TDenseSpaceType,
                     {
                         for (int i=0; i < SystemSize; i++)
                             b[i] = zwork(i,j);
-                        mpLinearSolver->Solve(Az,x,b);
+                        try {
+                            mpLinearSolver->PerformSolutionStep(Az,x,b);
```

Comment:
@pooyan-dadvand 
Here I am now able to catch the error if the function is not implemented and react accordingly